Features/Stories/Fieldtypes/AuthorsFieldType.feature

```diff
@@ -0,0 +1,180 @@
+Feature: Test the validations done on fields from PlatformUI - Authors fieldtype
+    In order to validate the authors fieldtype
+    As an Editor  user
+    I need to be able to create, update and delete content with authors fieldtypes
+
+    Background:
+        Given I am logged in as an Editor in PlatformUI
+
+    ##
+    # Validate the existence of expected fields from a field type when creating a content
+    ##
+    @javascript
+    Scenario: A Content of a Content Type that has an author fieldtype must have an author and an email fields
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this Content Type
+        Then I should have an "name" field
+        And I should have an "email" field
+
+    @javascript
+    Scenario: A Content of a Content Type that has an author fieldtype must be prefilled with user's name and email
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this Content Type
+        Then I should have an "name" field filled with Editor's Username
+        And I should have an "email" field filled with Editor's Email
+
+    @javascript
+    Scenario: A Content of a Content Type that has an author fieldtype must have a "Remove this author" and a "Add another author" buttons
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this Content Type
+        Then I should have an "Remove this author" button
+        And I should have an "Add another author" button
+
+    @javascript
+    Scenario: When editing a Content the label of an author field must have the same name of the field type from the respective Content Type
+        Given a Content Type with an "authors" with "Name" "Participants" exists

```

Comment:
again this with ... exists part looks a bit weird to me. but not an expert on this topic...